### ISSUE TITLE
Build new GATK

### ISSUE THREAD

**Issue by geoffjentry** [broadinstitute/cromwell#2196]
@droazen has provided us a GATK4 snapshot which we can use for profiling purposes in the new joint genotyping workflow. He's provided the following instructions for building. Build this jar and make sure there are no issues on our end.

git clone git@github.com:broadinstitute/gatk.git gatk
cd gatk
git checkout COMMIT_ID  (I'll provide this in a follow-up email)
./gradlew clean localJar
Jar will be in build/libs with a name like gatk-package-4.alpha.2-*-SNAPSHOT-local.jar
You'll also want to copy the gatk-launch script from the root of the clone and put it in the same directory that the GATK4 jar is copied to.
Note that this particular GATK4 jar will contain only public tools, not the protected tools like HaplotypeCaller.

**Comment by geoffjentry** [comment]
I spoke to @droazen and we'll need a 2nd jar as well, for the GATK protected (at least for now). The repo there is `broadinstitute/gatk-protected`. He will provide a commit hash later this evening

**Comment by geoffjentry** [comment]
From @droazen 

Alright, the commit to use to build the jar to run GenomicsDBImport (using the instructions above) is: d4d97fcbb59efd9acbf8fabca7361b59512755bb

The tool is passing integration tests at this point, and it is completely worth your while to profile the current version and see how it compares to the SelectVariants approach. It's worth mentioning that in the next week or so we will add one additional argument to the tool which might further help performance. You can track the status of this here: https://github.com/broadinstitute/gatk/issues/2613

Hand-off complete -- have a good weekend everyone!

**Comment by kcibul** [comment]
I've made the builds of both toolsets, updated the dockerfile (see dsde-pipelines branch kc_jg_turbocharge), built and pushed the docker image for our testing (kcibul/tiledb-with-gcloud:2.2.5-1492828987) in the JG WDL

**Comment by droazen** [comment]
We've updated gatk-protected to remove the JSON requirement for reading from GenomicsDB in `GenotypeGVCFs`, and also made it so that you can use the same jar to run both the `GenomicsDBImport` tool and `GenotypeGVCFs`. The commit you want is `8812780144561904e6e529c4673c3770076f543b` in `https://github.com/broadinstitute/gatk-protected`. Updated clone/build commands:

```
git clone git@github.com:broadinstitute/gatk-protected.git gatk-protected
cd gatk-protected
git checkout 8812780144561904e6e529c4673c3770076f543b
./gradlew clean localJar
Jar will be in build/libs with a name like gatk-protected-package-*-SNAPSHOT-local.jar
You can use this jar to run either GenomicsDBImport or GenotypeGVCFs (no need for a second jar)
```

We're still waiting on a PR from Intel to add the --batchSize argument to the GenomicsDBImport tool, which will be necessary to control memory usage with many samples, but otherwise we should be good to go.

**Comment by katevoss** [comment]
@droazen is this complete?

**Comment by droazen** [comment]
It's happened for the 11k run, but the changes to the joint genotyping workflow haven't been merged into dsde-pipelines/master yet.

**Comment by kcibul** [comment]
This has been handed over to @eitanbanks who has been building GATK and making dockers all day.  Closing
